docs/agents/sematext-agent/releasenotes.md

```diff
@@ -35,6 +35,51 @@ NULL
 
 -->
 
+# Version 3.0.0
+
+Date: July 25, 2023
+
+## New Features
+
+- SC-5833 [**Kubernetes Components**](https://kubernetes.io/docs/concepts/overview/components/) monitoring integration: We are excited to announce that Sematext Agent now supports monitoring of critical Kubernetes components, including **API Server**, **CoreDNS**, **kube-proxy**, **Scheduler**, and **Etcd**. Gain deeper insights into your Kubernetes cluster's performance with this powerful integration.
```

Comment:
@boratanrikulu let's add kubelet reports as well, you'll be notified by @yagogou as soon as we've update them.